frontend/src/component/menu/routes.ts

```diff
@@ -45,6 +45,13 @@ import { LazyProject } from 'component/project/Project/LazyProject';
 import { AdminRedirect } from 'component/admin/AdminRedirect';
 
 export const routes: IRoute[] = [
+    {
+        path: '/',
+        title: 'Unleash',
+        component: ProjectListNew,
+        type: 'protected',
+        menu: {},
+    },
```

Comment:
Interesting: was this not here before? Maybe because the redirect would take care of taking you to `/projects` or something?

Reply:
The redirect was set to render on the root path, which caused the loop to trigger. Because you can still navigate to the root by clicking the unleash logo, I set this as the default root path.

Reply:
Thanks! Does that mean that the URL will be the root URL, but that the rendered component is the project list?

Reply:
Yes, I suppose we could setup a redirect to: `/projects`

Reply:
I don't really mind, to be honest. It used to be that that page only rendered "all features", right? So if that's not useful anymore, then rendering projects instead might make sense? I don't have enough context to say what's the right thing to do here, and I suspect that exploring the solution space for this would take up a lot more time than is warranted for this PR, so I'm happy for you to make a choice here. If you think it's worth a wider discussion (it might be), then we should probably loop in the UX team et al., and make it its own task.